integration_tests/testsuite/driver.py

```diff
@@ -84,9 +84,9 @@ def _main():
             sys.exit(1)
 
         logging.debug('Deploying app!')
-        version, deploy_url = deploy_app.deploy_app(args.image, args.directory)
+        version = deploy_app.deploy_app(args.image, args.directory)
 
-    application_url = args.url or deploy_url
+    application_url = args.url or test_util.retrieve_url_for_version(version)
```

Comment:
This sort of makes me wonder if we should drop the deploy flag, and just treat a test run that gets passed an args.url as the same as no deploy. WDYT?

Basically if someone doesn't give us a URL, we deploy. If they do, we don't deploy.